src/frontend/src/components/public/PIAFormTabs/accuracyRetention/index.tsx

```diff
@@ -15,7 +15,13 @@ import ViewComments from '../../../common/ViewComment';
 import { PiaSections } from '../../../../types/enums/pia-sections.enum';
 import Radio from '../../../common/Radio';
 
-export const AccuracyCorrectionAndRetention = () => {
+interface AccuracyCorrectionAndRetentionProps {
+  hideViewComments: boolean;
```

Comment:
would it be better to have it as view comment and default to true. When the pdf section calls it we pass in false.

Reply:
I would 100% agree that we should pass on the function level with default true value, not in the root level. I have changed it!